### DISCUSSION TITLE
Whatsapp templates reload

### DISCUSSION THREAD

**Discussion by rickysb** [chatwoot/chatwoot#8566] [General]
So, since I had setup Chatwoot, I have added alot of templates in Whatsapp Manager in Meta console. But I am unsure how I get Chatwoot to refresh / pull the latest ones from Meta console. Any ideas?

**Comment by rickysb** [comment]
Found a solution.
I went to Rails Console here
https://www.chatwoot.com/docs/self-hosted/deployment/chatwoot-ctl/#running-rails-console

And then ran the command:
Channels::Whatsapp::TemplatesSyncSchedulerJob.perform_now()

Now, facing problem with pagination of the templates. Not all templates are visible.

**Comment by sojan-official** [comment]
covered in https://github.com/chatwoot/chatwoot/pull/12007
